tests/test_star_rsem.yml

```diff
@@ -4,13 +4,13 @@
     - test_star_rsem
   files:
     - path: results_test/differential_gene_expression/DE_genes_tables/DE_contrast_condition_cellline_H1_vs_GM12878.tsv
-      md5sum: f41854e3d6d69f0c6dc556dfd82b2874
+      md5sum: 96ae6d749099958313b30243f10dfaa1
```

Comment:
please just remove the md5sums if they are not constant over runs

Reply:
see above